src/components/docs/Buckets/Bucket/styles.module.css

```diff
@@ -0,0 +1,23 @@
+.root {
+  color: var(--doc-black);
+  transition: .15s ease-in-out;
+  width: var(--doc-bucket-size);
+}
+
+.root:hover {
+  box-shadow: 0px 10px 15px 0px rgba(0,0,0,0.2);
+  text-decoration: none;
+}
+
+.root h4 {
+  font-weight: normal;
+}
+
+/* Properties */
+.has-rounded-corners {
```

Comment:
We should be consistent with class name syntax. Looks like we are using camelCase most places